### PR TITLE
support load asset in internal storage or sdcard

### PR DESCRIPTION
**Pull request by codetypess**

### CODE REVIEW

File: cocos/ui/UIWebView/UIWebViewImpl-android.cpp [hunk 1]
```diff
@@ -142,15 +142,18 @@ std::string getUrlStringByFileName(const std::string &fileName) {
     // LOGD("error: %s,%d",__func__,__LINE__);
     const std::string basePath("file:///android_asset/");
     std::string fullPath = cocos2d::FileUtils::getInstance()->fullPathForFilename(fileName);
-    const std::string assetsPath("assets/");
-
     std::string urlString;
-    if (fullPath.find(assetsPath) != std::string::npos) {
-        urlString = fullPath.replace(fullPath.find_first_of(assetsPath), assetsPath.length(), basePath);
+    if (fullPath.c_str()[0] == '/') {
```
Comment (minggo) [inline cocos/ui/UIWebView/UIWebViewImpl-android.cpp]: - may crash if `fullPath` is empty
- `fullPath[0] == '/'` is better?

File: cocos/ui/UIWebView/UIWebViewImpl-android.cpp [hunk 2]
```diff
@@ -142,15 +142,18 @@ std::string getUrlStringByFileName(const std::string &fileName) {
...
+        urlString = "file://" + fullPath;
```
Comment (minggo) [inline cocos/ui/UIWebView/UIWebViewImpl-android.cpp]: `urlString.append("file://").append(fullPath)` is better?

File: cocos/ui/UIWebView/UIWebViewImpl-android.cpp [hunk 3]
```diff
@@ -142,15 +142,18 @@ std::string getUrlStringByFileName(const std::string &fileName) {
...
     } else {
-        urlString = fullPath;
+        if(fullPath.find("assets/") == 0) {
+            urlString = basePath + fullPath.substr(7);
```
Comment (minggo) [inline cocos/ui/UIWebView/UIWebViewImpl-android.cpp]: hardcode is not good.

File: cocos/ui/UIWebView/UIWebViewImpl-android.cpp (hunk 1 above)
Comment (codetypess) [reply cocos/ui/UIWebView/UIWebViewImpl-android.cpp]: yes, will better

**Review by minggo — APPROVED** [review]

**Pull request merged**